tests/testthat/test-estimate_dispersion.R

```diff
@@ -17,36 +17,46 @@ nowcast1 <- matrix(
     10, 7, 1,
     15, 12, 2,
     14, 16, 3,
-    11, 21, 8,
+    11, 21, 8.1,
     10, 15.3, 3.5
   ),
   nrow = 5,
   byrow = TRUE
 )
 nowcast2 <- matrix(
   c(
-    10, 7, 0.8,
-    15, 12, 2.0,
+    10, 7, 1,
+    15, 12, 2.3,
     14, 16, 2.5,
-    11, 25.5, 2
+    11, 25.5, 2.1
   ),
   nrow = 4,
   byrow = TRUE
 )
+nowcast3 <- matrix(
```

Comment:
These tests only make sense if at the time of implementation we understand what the code is doing so I think we need more unit tests for the internals here and a bit more clarity on what we expect the output to be?

Reply:
Can add to these some examples where we'd expect very high or low dispersion as a function of delay?